### PR TITLE
Use ruby 2.5.7 and install the same was as our ruby base image

### LINKED ISSUE #41: Update ruby version in image
The image is currently using ruby 2.3 which is EOL. We should update to 2.5.5 or later (2.5.5 is what container-ruby is currently at).

Maybe we should also change to run ruby-install with --system (just like https://github.com/ManageIQ/manageiq-appliance-build/pull/234)?

### PR DESCRIPTION
**Pull request by carbonin**
Fixes #41

### CODE REVIEW

**Review by Fryguy — APPROVED** [review]

**Comment by Fryguy** [comment]
See also https://github.com/ManageIQ/dbus_api_service/pull/16

File: Dockerfile [hunk 1]
```diff
@@ -67,22 +67,18 @@ RUN (cd /lib/systemd/system/sysinit.target.wants && for i in *; do [ $i == syste
...
-    curl -sL https://copr.fedorainfracloud.org/coprs/postmodern/ruby-install/repo/fedora-25/postmodern-ruby-install-fedora-25.repo -o /etc/yum.repos.d/ruby-install.repo && \
-    sed -i 's/\$releasever/25/g' /etc/yum.repos.d/ruby-install.repo
+    curl -sL https://copr.fedorainfracloud.org/coprs/manageiq/ManageIQ-Master/repo/epel-7/manageiq-ManageIQ-Master-epel-7.repo -o /etc/yum.repos.d/manageiq.repo
 
 # Install ruby-install and make
 RUN yum -y install --setopt=tsflags=nodocs ruby-install make
 
-# Install Ruby 2.3.1
-RUN ruby-install ruby 2.3.1 -- --disable-install-doc && rm -rf /usr/local/src/* && yum clean all
+RUN ruby-install --system ruby 2.5.5 -- --disable-install-doc --enable-shared && rm -rf /usr/local/src/* && yum clean all
```
Comment (Fryguy) [inline Dockerfile]: Should this be 2.5.7?

Comment (carbonin) [reply Dockerfile]: I just used what was currently in container-ruby. We can go right to 2.5.7 if you want though.

**Comment by simaishi** [comment]
Thanks for the PR @carbonin I should have created it myself, but I just wasn't sure how to test this image standalone!

As for version... I'm thinking we can stay at 2.5.5 as that's what appliances and other container images are currently using, and we have no branch specific build for this container image.

**Comment by carbonin** [comment]
No problem @simaishi I needed the Hacktoberfest internet points :laughing:

**Comment by carbonin** [comment]
As for testing I went and opened a PR to change the travis version of the dbus API before seeing that @jrafanie had done it already. I figure that's good enough.

I'm also looking at these images for a separate effort so they'll get some testing in the next few days.

File: Dockerfile (hunk 1 above)
Comment (jrafanie) [reply Dockerfile]: I think we want upstream master to always have the latest release of our supported rubies... we'll always be behind but it's worth getting the latest when we can...

and release branches will narrow down to the single version we're supporting...

**Comment by jrafanie** [comment]
> No problem @simaishi I needed the Hacktoberfest internet points 😆

@carbonin before it's HacktOVERfest . 🤣

File: Dockerfile (hunk 1 above)
Comment (carbonin) [reply Dockerfile]: Does latest here mean moving to 2.6? Or are you still just talking about 2.5.7?

Comment (simaishi) [reply Dockerfile]: > I think we want upstream master to always have the latest release of our supported rubies... we'll always be behind but it's worth getting the latest when we can...
> and release branches will narrow down to the single version we're supporting...

Agreed, it's just this repo and some other container image repos only have `master` branch.

**Comment by carbonin** [comment]
Updated container-ruby https://github.com/ManageIQ/container-ruby/pull/14

File: Dockerfile (hunk 1 above)
Comment (jrafanie) [reply Dockerfile]: I think 2.5.7 is the master supported version with 2.6.5 being the one we're slowly moving to.  We make the rules so we can decide how we want to do it.  

But, yes, 2.5.7 seems like the best choice here.

**Pull request merged**